CactusCareApi/CactusCare.BLL/Validators/LoginDtoValidator.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CactusCare.Abstractions.DTOs;
+using FluentValidation;
+
+namespace CactusCare.BLL.Validators
+{
+    public class LoginDtoValidator : AbstractValidator<LoginDto>
+    {
+        public LoginDtoValidator()
+        {
+            RuleFor(l => l.UserName).NotEmpty().Length(4, 20).Matches(@"^[a-zA-Z0-9]+([_ -]?[a-zA-Z0-9])*$");
+            RuleFor(l => l.Password).NotEmpty().Length(8, 20)
```

Comment:
I am not sure password validation matches rules in Identity configuration, so either change identity configuration or change password validation